weblate/checks/tests/test_chars_checks.py

```diff
@@ -217,6 +218,21 @@ def test_eu(self) -> None:
         self.do_test(False, ("Text!", "¡Texte!", ""), "eu")
```

Comment:
Please also add tests to `EndQuestionCheckTest` and `EndExclamationCheckTest` to verify interrobang skipping.